SingleViewApi.Tests/V1/UseCase/GetCustomerByIdUseCaseTests.cs

```diff
@@ -90,7 +92,7 @@ public void GetsACustomerFromPersonApiJigsawAndAcademyCouncilTax()
                 }
             });
 
-            var fakeContactDetails = _fixture.CreateMany<ContactDetails>().ToList();
+            var fakeContactDetails = new List<ContactDetails>() { _fixture.Create<ContactDetails>() };
```

Comment:
is this different to the previous way?

Reply:
just adds one, and makes it less confusing to test